### ISSUE TITLE
API rate limit exceeded - github

### ISSUE THREAD

**Issue by yasindagasan** [ultralytics/yolov5#3134]
## 🐛 Bug

I was running the code for 85 folders and just came across with the below issue. I printed the response and I got the following error: it basically says that the API rate limit was exceeded. Any solution rather than at which folder the detections got stuck and resuming from there? This happened several times today. I cannot update the repo at the moment as I feel like I have to re-train all my models using the recent network architecture.

## To Reproduce

Input: 
```
Namespace(agnostic_nms=True, augment=True, classes=None, conf_thres=0.5, device='', exist_ok=True, img_size=1280, iou_thres=0.3, name='a', project='/media/dagasan/b9faeee7-8832-4b01-80f9-a9dae2314657/projects/x', save_conf=True, save_txt=True, source='/media/dagasany/b9faeee7-8832-4b01-80f9-a9dae2314657/projects/y/folder_1_txt', update=False, view_img=False, weights=['../../02-models/yolov5/task/task_1.pt'])

```

Output:
```

YOLOv5 🚀 2021-3-28 torch 1.7.1 CUDA:0 (GeForce RTX 2080 Ti, 11016.9375MB)

**{'message': "API rate limit exceeded for --my ip-- (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", 'documentation_url': 'https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting'}**
Traceback (most recent call last):
  File "/media/dagasan/b9faeee7-8832-4b01-80f9-a9dae2314657/01-codes/yolov5-master-23032021/utils/google_utils.py", line 28, in attempt_download
    assets = [x['name'] for x in response['assets']]  # release assets, i.e. ['yolov5s.pt', 'yolov5m.pt', ...]
KeyError: 'assets'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../yolov5-master-23032021/detect.py", line 175, in <module>
    detect()
  File "../yolov5-master-23032021/detect.py", line 33, in detect
    model = attempt_load(weights, map_location=device)  # load FP32 model
  File "/media/dagasan/b9faeee7-8832-4b01-80f9-a9dae2314657/01-codes/yolov5-master-23032021/models/experimental.py", line 117, in attempt_load
    attempt_download(w)
  File "/media/dagasan/b9faeee7-8832-4b01-80f9-a9dae2314657/01-codes/yolov5-master-23032021/utils/google_utils.py", line 32, in attempt_download
    tag = subprocess.check_output('git tag', shell=True).decode().split()[-1]
IndexError: list index out of range
```

## Expected behavior
It gets stuck around 60th folder and I cannot run the code for any detection task for a while. It then starts to work again.

## Environment

 - OS: [e.g. Ubuntu 21.04]
 - GPU [e.g. 2080 Ti]

**Comment by glenn-jocher** [comment]
This was fixed several months ago.

Update your code.

**Comment by yasindagasan** [comment]
okay great! thanks for the prompt response.

**Comment by glenn-jocher** [comment]
@yasindagasan good news 😃! Your original issue may now been fixed ✅ in PR #1894 from January 10th, 2021.  To receive this update you can:
- `git pull` from within your `yolov5/` directory
- `git clone https://github.com/ultralytics/yolov5` again
- Force-reload [PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/): `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- View our updated notebooks:  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
